File: Arjuna/versions/PR3.3_NT/initial/tests/uiduniqueness/uiduniqueness.cxx
/*
 * Copyright (C) 1993
 * 
 * Department of Computing Science,
 * The University,
 * Newcastle upon Tyne,
 * UK.
 */

/*
 * $Id: UidUniqueness.cc,v 1.2 1994/09/14 08:56:22 ngdp Exp $
 */

#include <System/iostream.h>
#include <Common/Uid.h>

int main(int, char*[])
{
    Uid uids[10];

    for (int i = 0; i < 10; i++)
	cout << uids[i] << '\n' << flush;

    return 0;
}

File: ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/uid/UidUniqueness.java
package com.hp.mwtests.ts.arjuna.uid;

/*
 * Copyright (C) 1998, 1999, 2000, 2001,
 *
 * Arjuna Solutions Limited,
 * Newcastle upon Tyne,
 * Tyne and Wear,
 * UK.
 *
 * $Id: UidUniqueness.java,v 1.2 2003/06/19 10:51:22 nmcl Exp $
 */

import com.arjuna.ats.arjuna.common.*;

public class UidUniqueness
{
    
public static void main (String[] args)
    {
	for (int i = 0; i < 100; i++)
	{
	    Uid u = new Uid();
	    System.out.println(u + " " + u.hashCode());
	}
    }
    
}
